### ISSUE TITLE
Fullscreen video to hide sidebar/vertical tabs

### ISSUE THREAD

**Issue by StickNik** [samihaddad/vertical-tabs-chrome-extension#21]
Early on I was fullscreening a YouTube video and was surprised that the vertical tabs stayed visible, and expected that the sidebar would be covered or hidden. It seems to be an inherent feature of Chrome sidebar as far as I can tell

Given how easy it is to toggle the sidebar I haven't minded it, and now I don't necessarily want to hide the sidebar without choosing to. Having said that, I'm sure some people would always want when a video is fullscreen for vertical tabs/the sidebar to become hidden like in MS Edge or Brave, so it's worth considering it as an option if it's possible to have a fullscreen event trigger the sidebar to be hidden, unless that's possibly something that should be a chromium issue or option.

**Comment by samihaddad** [comment]
Yes this is the default behavior of the sidebar and there doesn't seem to be an option to change it, not even in an extension. That said, it would be a good idea to report this to Chrome directly, as it makes more sense to be part of the browser settings.
